### PR TITLE
add incident type schema

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
Ready

## Description
1. Added schema for incident types.
2. Added fieldMapping for playbook schema.
3. Fixed cliName issue.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

**Comment by orlichter1** [comment]
Checking new build here: https://github.com/demisto/content/pull/5382

File: demisto_sdk/common/hook_validations/incident_field.py [hunk 1]
```diff
@@ -38,7 +38,8 @@ def is_valid_name(self):
         """Validate that the name and cliName does not contain any potential incident synonyms."""
         name = self.current_file.get('name', '')
         cli_name = self.current_file.get('cliName', '')
-        bad_words = {'incident', 'case', 'alert', 'event', 'play', 'ticket', 'issue'}
+        bad_words = {'incident', 'case', 'alert', 'event', 'playbook', 'ticket', 'issue',
+                     'incidents', 'cases', 'alert', 'events', 'playbooks', 'tickets', 'issues'}
```
Comment (ronykoz) [inline demisto_sdk/common/hook_validations/incident_field.py]:
```suggestion
                     'incidents', 'cases', 'alerts', 'events', 'playbooks', 'tickets', 'issues'}
```

File: demisto_sdk/validation/file_validator.py [hunk 2]
```diff
@@ -21,7 +21,7 @@
     IMAGE_REGEX, TEST_PLAYBOOK_REGEX, DIR_LIST_FOR_REGULAR_ENTETIES, \
     PACKAGE_SUPPORTING_DIRECTORIES, YML_BETA_INTEGRATIONS_REGEXES, PACKAGE_SCRIPTS_REGEXES, YML_INTEGRATION_REGEXES, \
     PACKS_DIR, PACKS_DIRECTORIES, Errors, PLAYBOOKS_REGEXES_LIST, JSON_INDICATOR_AND_INCIDENT_FIELDS, PLAYBOOK_REGEX, \
-    JSON_ALL_LAYOUT_REGEXES, REPUTATION_REGEX, CHECKED_TYPES_REGEXES
+    JSON_ALL_LAYOUT_REGEXES, REPUTATION_REGEX, CHECKED_TYPES_REGEXES, INCIDENT_TYPE_REGEX
```
Comment (ronykoz) [inline demisto_sdk/validation/file_validator.py]:
```suggestion
    JSON_ALL_LAYOUT_REGEXES, REPUTATION_REGEX, CHECKED_TYPES_REGEXES
```

**Comment by orlichter1** [comment]
Passed Van Dijks build. 
image
waiting for final answer from Bar Saar about wether to accept fromVersion or not.
image

**Pull request merged**